docs/pages/versions/unversioned/bare/updating-your-app.md

```diff
@@ -0,0 +1,173 @@
+
+### Release Channels
+
+Because the current implementation of the Expo updates service relies heavily on SDK version (a managed-workflow concept), if you're using `expo publish` you cannot yet use Runtime Version to manage compatibility of your updates and binaries. Instead, you can use [release channels](../../distribution/release-channels/). A typical workflow would be to create a new release channel for each new binary you build (or at least every new binary with an incompatible change in the native-JavaScript interface) by publishing to that new channel with `expo publish --release-channel <channel-name>`. After creating a build with this release channel name configured, you can continue to publish future updates to this same release channel as long as they remain compatible with that build. Only builds that were configured to use that release channel will receive those updates.
+
+### Statically Hosted Updates
+
+Since headers sent in requests by `expo-updates` do not affect statically hosted updates (such as update packages created by `expo export`), you must host incompatible updates at different static URLs in order to control compatibility.
+
+## Embedding Assets
+
+In addition to loading updates from remote servers, apps with `expo-updates` installed also include the necessary capability to load updates embedded in the app binary. This is critical to ensure that your app can load for all users immediately upon installation, without needing to talk to a server first.
+
+Updates embedded into your app binary look slightly different from how vanilla React Native apps embed JavaScript and assets. In particular, updates from `expo-updates` include the manifest metadata along with all of the assets required to run the update. The manifest should be in a file called `app.manifest` and the JavaScript bundle in a file called `app.bundle`. For iOS, these should be added to your Xcode project so they are bundled into the IPA. For Android, placing them in the `android/app/src/main/assets` folder is sufficient for gradle to bundle them into the APK.
+
+Assets that you `require` in your JavaScript source can be embedded into your app binary. If you use `expo-asset` in your project (included by default if you have the `expo` package installed), you can control which `require`d assets will be bundled into your binary using the [assetBundlePatterns](../../workflow/configuration/) key in `app.json` to provide a list of paths in your project directory:
+
+```
+"assetBundlePatterns": [
+  "**/*" // or "assets/images/*", etc.
+],
+```
+
+Assets with paths matching the given patterns will be bundled into your native binaries next time you run `expo build`. If you have a particularly large asset, such as a video, that should be downloaded at runtime rather than when the app is installed, you can use `assetBundlePatterns` to exclude it while still `require`ing it in your JavaScript code.
```

Comment:
Large videos shouldn't be downloaded as part of an OTA update service and should be embedded in the application.